db-run.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
```

Comment:
I think you should `set -e` here. There's no point in continuing if this breaks (is there?).